src/abi/typeFormula.ts

```diff
@@ -1,18 +1,21 @@
 export class TypeFormula {
     name: string;
+    metadata: any;
```

Comment:
maybe change type to `string` here, as well

Reply:
here we will let any because is the general object

Reply:
Don't you think it would be better to use `object` type instead?

Reply:
it can be also a primitive that's why I think any is a better option here